src/compiler/Parser/ParsedHierarchy/Declaration/ExternalDeclaration.ts

```diff
@@ -15,6 +15,10 @@ export class ExternalDeclaration extends ParsedObject implements INamedContent {
     super();
   }
 
+  get typeName(): string {
+    return "EXTERNAL";
```

Comment:
Thinking about it some more, I guess it looks better that way that `GetType().Name` from upstream. 👍

We should document those differences, in case someone else looks at the code down the line. It's a bit tedious, but… it's helpful in the long run!